### DISCUSSION TITLE
Does this lib supports JVM desktop applications?

### DISCUSSION THREAD

**Discussion by MessiasLima** [GitLiveApp/firebase-kotlin-sdk#329] [Q&A] [answered]
Hello, I wanted to use it in a KMP application that has a desktop JVM module. Is it possible?

**Comment by nbransby** [comment, marked as answer]
By itself it doesn't but we have written a port of the android firebase sdk that works on a vanilla JVM but haven't open sourced it yet as it remains very rough around the edges

It only supports firestore, database and functions and a small subset of Auth.

**Reply by MessiasLima** [reply]
Thank you for your reply. I hope to be able to contribute when it is open sourced

**Reply by nbransby** [reply]
done https://github.com/GitLiveApp/firebase-java-sdk

**Reply by MessiasLima** [reply]
Thank you!
